### PR TITLE
[16.0][MIG] maintenance_security: Migration to version 16.0

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
- Standard migration to v16

TT45871

@Tecnativa

@pedrobaeza @victoralmau

### CODE REVIEW

File: maintenance_security/tests/test_maintenance_security.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        cls.env = cls.env(
+            context=dict(
+                cls.env.context,
+                mail_create_nolog=True,
+                mail_create_nosubscribe=True,
+                mail_notrack=True,
+                no_reset_password=True,
+                tracking_disable=True,
+            )
+        )
```
Comment (victoralmau) [inline maintenance_security/tests/test_maintenance_security.py]:
```suggestion
from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT


class TestMaintenanceSecurity(common.TransactionCase):
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, **DISABLED_MAIL_CONTEXT))
```

**Review by victoralmau — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Any advance in the CI question?

/ocabot migration maintenance_security

**Comment by etobella** [comment]
@carolinafernandez-tecnativa Can you reviews the tests?

**Comment by carolinafernandez-tecnativa** [comment]
> @carolinafernandez-tecnativa Can you reviews the tests?

I have checked and the problem is with CI and not with module tests. I still not being able to solve CI problem.

**Comment by etobella** [comment]
Can you rebase, I think it is related to another issue and rebasing it might fix it :smile:

**Comment by carolinafernandez-tecnativa** [comment]
> Can you rebase, I think it is related to another issue and rebasing it might fix it 😄

I rebased but still the same CI issue

**Comment by etobella** [comment]
I see... it is related to https://github.com/odoo/odoo/pull/145800

With the change it was fixed for me :smile:

**Review by etobella — APPROVED** [review]
The error was general from odoo. Now it is fixed :tada: 

/ocabot merge nobump

**Pull request merged**